### ISSUE TITLE
The problem of cerr viewer

### ISSUE THREAD

**Issue by mricenter** [cerr/CERR#191]
hello, every,

I had met some troubles with the CERR software. Could you help me with that?

The version of my MATLAB is 2014b.

Best regards

-----------------------------------------------------------------
Attempt to reference field of non-structure array.
Error in putFileMenu (line 37)
    hMenu = uimenu(stateS.handle.CERRSliceViewer, 'label', '&File',
    'callback', 'putFileMenu', 'Interruptible', 'off');

Error in sliceCallBack (line 236)
        stateS.handle.CERRFileMenu          = putFileMenu(hCSV);
Error in CERR (line 156)
            sliceCallBack('init');
 
Error while evaluating UIControl Callback
-----------------------------------------------------------------

**Comment by adityaapte** [comment]
Could you use a newer (2016b or later) MATLAB version?

**Comment by mricenter** [comment]
No, did you mean that problem may be associated with MATLAB version?

**Comment by maacke1** [comment]
I had the same issue using 2014a

**Comment by maacke1** [comment]
no problem with previous R2008 though.
